### PR TITLE
test: add missing artii cert-key pair

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by nebril** [comment]
test-only-with-quarantined --focus="K8sPolicyTest"

**Comment by nebril** [comment]
test-only-with-quarantined --focus="K8sPolicyTest"

**Comment by nebril** [comment]
test-only-with-quarantined --focus="K8sPolicyTest"

File: test/k8sT/manifests/internal-artii.crt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```
Comment (aanm) [inline test/k8sT/manifests/internal-artii.crt:1]: I think it's important to document wow were these certs created.

**Review by aanm — CHANGES_REQUESTED** [review]

**Comment by nebril** [comment]
test-only-with-quarantined --focus="K8sPolicyTest"

File: test/k8sT/manifests/internal-artii.crt (hunk 1 above)
Comment (nebril) [reply test/k8sT/manifests/internal-artii.crt:1]: @aanm Should this be documented in the commit message? I finally found the key to the root key in a commit msg by Jarno :D

Comment (aanm) [reply test/k8sT/manifests/internal-artii.crt:1]: Yeah commit message would be good IMO

**Comment by nebril** [comment]
test-only-with-quarantined --focus="K8sPolicyTest"

**Comment by nebril** [comment]
test-4.9

**Review by aanm — APPROVED** [review]

**Comment by nebril** [comment]
test-only-with-quarantined --focus="K8sPolicyTest Basic Test"

**Comment by nebril** [comment]
test-only-with-quarantined --focus="K8sPolicyTest.Basic.Test"

**Comment by nebril** [comment]
Tests passed, this will fix quarantine pipelines, merging.

**Pull request merged**